### PR TITLE
Add Arkansas Tech University

### PR DESCRIPTION
**Pull request by AnalogCyan**
Add Arkansas Tech University as they are now fully implementing 2FA.

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]
SVG logos are preferred. Please search for one and if you can't find any, use a PNG image that is exactly 32x32 pixels, doesn't contain any text and less than 2KiB in file size.

**Comment by 40504** [comment]
@AnalogCyan look at this :), enjoy

**Comment by AnalogCyan** [comment]
@40504 nice, thanks!

**Review by phallobst — APPROVED** [review]

File: entries/a/atu.edu.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "custom-software",
+      "custom-hardware"
+    ],
+    "custom-software": [
+      "Duo"
+    ],
+    "documentation": "https://support.atu.edu/support/solutions/folders/7000078509",
+    "keywords": [
+      "education"
+    ]
```
Comment (ApCoder123) [inline entries/a/atu.edu.json:18]:
```suggestion
    ],
    "regions": [
      "us"
    ]
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**